graph/gen_backend_ops.go

```diff
@@ -912,7 +913,7 @@ func (ni *nodeInputsDiv) String() string {
 	)
 }
 
-// Div returns the element-wise subtraction of the two values.
+// Div returns the element-wise division of the two values.
```

Comment:
Sry, this file is auto-generated, I had forgotten. Everything starting with "gen_" is auto-generated.

We'll have to to fix this on the source -- I can do it later.